sidebars.js

```diff
@@ -162,6 +192,7 @@ module.exports = {
 				'standards/version-control-conventions',
 				'standards/productivity-tips-and-tricks',
 				'standards/cheatsheets',
+                'standards/yoast-seo-product-sheet',
```

Comment:
Indentation seems off.

Reply:
Can you wrangle this from your side? Think it's a Windows thing; struggling to align!